_posts/patterns/2015-04-15-icons.md

```diff
@@ -126,29 +126,29 @@ info:               Icons are used to clearly and visually represent some inform
             </span>
         </div>
         <div class="icon-classname">
-            Stop
+            stop
         </div>
     </div>
     <div class="example example-icon">
         <div class="icon-display">
             <span class="icon-fallback icon-fallback-text">
-                <span class="icon icon-arrows-alt" aria-hidden="true"></span>
+                <span class="icon icon-fullscreen" aria-hidden="true"></span>
```

Comment:
I went back-and-forth on this when I added it. It seems that fullscreen is the convention used for the mode:

http://english.stackexchange.com/questions/162421/fullscreen-or-full-screen